### PR TITLE
Replace ruby sass with npm sass

### PR DESCRIPTION
**Pull request by musangowope**
- Install sass package
- remove build.sh
- add build command for package.json
- update build sass step in readme.md
resolves https://github.com/laws-africa/indigo-web/issues/20

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "/scss"
   ],
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "set -e && set -x && sass --style compressed scss/indigo-web.scss css/indigo-web.min.css && sass scss/indigo-web.scss css/indigo-web.css"
```
Comment (musangowope) [inline package.json:12]: the syntax of the command is from `build.sh`
I ran it and it does generate the CSS,
just not sure if I should still in the `set -e` and `set -x`. Not sure what that does

Comment (longhotsummer) [reply package.json:12]: The `set` commands tell bash to fail if a command fails, but github actions effectively does this, so you can drop them.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,347 @@
+{
+  "name": "indigo-web",
+  "version": "5.2.1",
```
Comment (longhotsummer) [inline package-lock.json:3]: This is generated with node 17, which we're not using yet.

You'll need to delete this, ensure you're using node 16 (which is still the recommended version as per https://nodejs.org/en/) and then re-generate the lock file.

File: README.md [hunk 3]
```diff
@@ -44,8 +44,7 @@ their values.
 
 To build changes:
 
-* gem install sass
-* ./build.sh
+* npm run build
```
Comment (longhotsummer) [inline README.md:47]: Please include `* npm install`

**Review by longhotsummer — CHANGES_REQUESTED** [review]
Looks good, minor feedback on node versions and readme.

File: package.json (hunk 1 above)
Comment (musangowope) [reply package.json]: cool, removed👍🏽

File: package-lock.json (hunk 2 above)
Comment (musangowope) [reply package-lock.json:3]: ok. Hmm, its bit strange. I checked in my terminal and I am using node 16. However, nevertheless I did follow your advice anyway just to be doubly sure. My actions:
- `nvm use 16`
- delete `package-lock.json`
- remove `npm install`
Still produces the same lock file (no difference in git)🤔

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**